tests/out/msl/control-flow.msl

```diff
@@ -99,6 +103,7 @@ kernel void main_(
         }
         case 3: {
             pos = 2;
+            return;
```

Comment:
I was completely mystified by this new `return` until I found `proc::terminator::ensure_block_returns`...

Reply:
Haha, yeah, it took me a while to find out about `ensure_block_returns` as well.